backend/src/main/java/com/darass/darass/user/service/UserService.java

```diff
@@ -36,4 +38,13 @@ public void deleteById(Long id) {
         users.deleteById(id);
     }
 
+    public PasswordCheckResponse checkGuestUserPassword(PasswordCheckRequest passwordCheckRequest) {
+        Optional<User> expectedUser = users.findById(passwordCheckRequest.getGuestUserId());
```

Comment:
```suggestion
        Optional<User> possibleUser = users.findById(passwordCheckRequest.getGuestUserId());
```
우기랑 같은 피드백을 남겼는데, Optional 변수명은 "possible + Type" 방식으로 쓰는 것이 어떨지 이야기하는 포스팅이 있었어요. 저는 더 직관적이라고 생각이 드는데 아론은 어떤가요?
https://betterprogramming.pub/useful-tips-for-naming-your-variables-8139cc8d44b5

Reply:
좋은 지적 감사합니다~ 반영했습니다!